### PR TITLE
add global batchnorm unittest

### PR DESCRIPTION
**Pull request by hjchen2**

### CODE REVIEW

File: python/oneflow/test/modules/test_consistent_batchnorm.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    return y
+
+
+class TestBatchNormModule(flow.unittest.TestCase):
+    @globaltest
+    def test_batchnorm(test_case):
+        for placement in all_placement():
+            # mean and variance maybe inconsistent if input is split into each rank for training.
+            # max_dim=0 will disable generating split sbp.
+            for sbp in all_sbp(placement, max_dim=0):
```
Comment (clackhan) [inline python/oneflow/test/modules/test_consistent_batchnorm.py:71]: 这里是不是设置```except_split=True```就可以了

File: python/oneflow/test/modules/test_consistent_batchnorm_add_relu.py [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+    params_sbp = [flow.sbp.broadcast for _ in range(len(sbp))]
+    fused_bn = flow.nn.FusedBatchNorm2d(
+        channel, track_running_stats=track_running_stats
+    ).to_global(placement=placement, sbp=params_sbp)
+    fused_bn.weight = flow.nn.Parameter(
+        weight.to_global(placement=placement, sbp=params_sbp)
+    )
+    fused_bn.bias = flow.nn.Parameter(
+        bias.to_global(placement=placement, sbp=params_sbp)
+    )
```
Comment (clackhan) [inline python/oneflow/test/modules/test_consistent_batchnorm_add_relu.py:46]: 第38行已经执行过to_global了，这里还有必要再次分别to_global吗

File: python/oneflow/test/modules/test_consistent_batchnorm_add_relu.py [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
```
Comment (clackhan) [inline python/oneflow/test/modules/test_consistent_batchnorm_add_relu.py:140]: 同上

File: python/oneflow/test/modules/test_consistent_bernoulli.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    @globaltest
+    def test_bernoulli(test_case):
+        for placement in all_placement():
+            # bernoulli only has cpu kernel
+            if placement.type != "cpu":
+                continue
+            ndim = random(1, 4).to(int).value()
+            for sbp in all_sbp(placement, max_dim=min(2, ndim)):
+                _test_bernoulli(test_case, ndim, placement, sbp)
+                _test_bernoulli_with_generator(test_case, ndim, placement, sbp)
```
Comment (clackhan) [inline python/oneflow/test/modules/test_consistent_bernoulli.py:60]: 这个为什么没有用autotest呢，pytorch也有这个接口

File: python/oneflow/test/modules/test_consistent_fused_tril_softmax_mask_scale.py [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        arg_dict["seq_length"] = [8, 16]
+        arg_dict["channel"] = [8, 16]
+        arg_dict["p"] = [0.0, 1.0]
+        arg_dict["diagonal"] = [0, 1, 2]
+        arg_dict["tril_scale_value"] = [2, 4, 10]
+
+        for arg in GenArgList(arg_dict):
+            for placement in all_placement():
+                if placement.type != "cuda":
+                    continue
```
Comment (clackhan) [inline python/oneflow/test/modules/test_consistent_fused_tril_softmax_mask_scale.py:78]: 这个为什么要跳过cuda呢？

File: python/oneflow/test/modules/test_consistent_batchnorm.py (hunk 1 above)
Comment (hjchen2) [reply python/oneflow/test/modules/test_consistent_batchnorm.py:71]: 是一样的，不过我还是改一下，`except_split=True`更易懂一些

File: python/oneflow/test/modules/test_consistent_batchnorm_add_relu.py (hunk 2 above)
Comment (hjchen2) [reply python/oneflow/test/modules/test_consistent_batchnorm_add_relu.py:46]: 是没必要～～

File: python/oneflow/test/modules/test_consistent_bernoulli.py (hunk 4 above)
Comment (hjchen2) [reply python/oneflow/test/modules/test_consistent_bernoulli.py:60]: 因为随机出来的结果不同，无法和PyTorch对齐的

File: python/oneflow/test/modules/test_consistent_fused_tril_softmax_mask_scale.py (hunk 5 above)
Comment (hjchen2) [reply python/oneflow/test/modules/test_consistent_fused_tril_softmax_mask_scale.py:78]: 很多fuse只支持了cuda，这个也一样

File: python/oneflow/test/modules/test_consistent_batchnorm_add_relu.py (hunk 2 above)
Comment (hjchen2) [reply python/oneflow/test/modules/test_consistent_batchnorm_add_relu.py:46]: oo 这里是为了和local对齐而重新设置module的weight和bias，这里就不改了

File: python/oneflow/test/modules/test_consistent_batchnorm.py (hunk 1 above)
Comment (hjchen2) [reply python/oneflow/test/modules/test_consistent_batchnorm.py]: done

File: python/oneflow/test/modules/test_consistent_bernoulli.py (hunk 4 above)
Comment (clackhan) [reply python/oneflow/test/modules/test_consistent_bernoulli.py:60]: 好的

File: python/oneflow/test/modules/test_consistent_fused_tril_softmax_mask_scale.py (hunk 5 above)
Comment (clackhan) [reply python/oneflow/test/modules/test_consistent_fused_tril_softmax_mask_scale.py:78]: 好的

**Review by clackhan — APPROVED** [review]
